pkg/policy/identifier.go

```diff
@@ -42,3 +42,19 @@ type Endpoint interface {
 	GetSecurityIdentity() *identity.Identity
 	PolicyRevisionBumpEvent(rev uint64)
 }
+
+// EndpointSet is used to be able to group together a given set of Endpoints
+// that need to have a specific operation performed upon them (e.g., policy
+// revision updates).
+type EndpointSet struct {
+	Mutex     lock.RWMutex
```

Comment:
I would prefer for the Mutex to not be exposed but instead having `Add()`, `Delete()`, and `ForEach()` functions which hides the locking.

Reply:
@tgraf done